packages/re/re.1.7.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "jerome.vouillon@pps.univ-paris-diderot.fr"
```

Comment:
Yeah, I've been doing most of the maintenance work lately. But there's a small team of us doing this, so it would be best for me to consult with them if I should pick up the opam maintenance from now.